example_code/basic_examples/test/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "stylus-test"
```

Comment:
I would rename the package name to test-example. This might cause confusion with the stylus-test package from the SDK.